README.md

```diff
@@ -102,6 +102,32 @@ After deploying the model endpoint, you can access and test it by using the scri
 python examples/inference/api_server_simple/query_single.py --model_endpoint http://127.0.0.1:8000/gpt2
 ```
 
+## Getting Started With Docker
+This guide will assist you in setting up LLM-on-Ray on With Docker.
+
+#### 1. Build docker with dockerfile
+Software requirement: Ubuntu and Docker
+```bash
+git clone https://github.com/intel/llm-on-ray.git
+cd llm-on-ray
+## You can change any settings here 
+source dev/scripts/start_with_docker.sh
+## Docker flie path is 'dev/docker/Dockerfile.{DOCKER_FILE_NAME}'.
+build_and_prune #{YOUR_CONTAINER_NAME} #{DOCKER_FILE_NAME}
+start_docker #{YOUR_CONTAINER_NAME} #{YOUR_SOURCE_CODE_PATH} #{YOUR_MODEL_CACHE_PATH}
+```
+
+#### 2. Start Ray
+```bash
+docker exec #{YOUR_CONTAINER_NAME} bash -c "./dev/scripts/start-ray-cluster.sh"
```

Comment:
this script is inside the container, right? It's better to not to use a relative path. You can consider add all the scripts to PATH and call them directly